.github/workflows/python-package.yml

```diff
@@ -10,14 +10,26 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
```

Comment:
Okay, it's not part of this workflow here. We might want to check more than one Python version, however.

Reply:
Okay, you meant the same here; ignore my comment before then. Will add other python versions as well 😃

Reply:
Hi Janine, do we really need more linting checks on other versions? I used pymatgen and atomate2 as reference , and in there only one version is tested.

Reply:
okay, let's leave it.